tests/end_to_end/test_vulnerability_detector/cases/test_vulnerability.yaml

```diff
@@ -1,483 +1,298 @@
 
 - case: 'Upgrade: Cease vulnerability'
   id: upgrade_package_remove_vulnerability
   description: |
     Upgrade of a vulnerable which cease to be vulnerable
-    macos:
-      Used Package: http-proxy 0.7.0 - npm Format
-      "CVE": [],
-    windows:
-      Used Package: Node 18.20.0 - .msi Format
-      "CVE": [],
-    ubuntu:
-      Used Packages: Grafana 9.4.17 - .deb Format
-      CVE: [],
-    centos:
-      Used Packages: Grafana 9.4.17 - .rpm Format
-      CVE: [],
-  preconditions: null
   body:
-    tasks:
-      - operation: update_package
-        target: agent
-        check:
-          alerts: true
-          state_index: true
-        package:
-          from:
-            centos:
-              amd64: grafana-9.2.0-1
-              arm64v8: grafana-9.2.0-1
-            ubuntu:
-              amd64: grafana-9.2.0
-              arm64v8: grafana-9.2.0
-            windows:
-              amd64: node-v18.1.0
-            macos:
-              amd64: http-proxy-0.5.10
-              arm64v8: http-proxy-0.5.10
-          to:
-            centos:
-              amd64: grafana-9.4.17-1
-              arm64v8: grafana-9.4.17-1
-            ubuntu:
-              arm64v8: grafana-9.4.17
-              amd64: grafana-9.4.17
-            windows:
-              amd64: node-v18.20.0
-            macos:
-              amd64: http-proxy-0.7.0
-              arm64v8: http-proxy-0.7.0
+    operation: update_package
+    package:
+      from:
+        centos:
+          amd64: grafana-9.2.0-1
+          arm64v8: grafana-9.2.0-1
+        ubuntu:
+          amd64: grafana-9.2.0
+          arm64v8: grafana-9.2.0
+        windows:
+          amd64: node-v18.1.0
+        macos:
+          amd64: http-proxy-0.5.10
+          arm64v8: http-proxy-0.5.10
+      to:
+        centos:
+          amd64: grafana-9.4.17-1
+          arm64v8: grafana-9.4.17-1
+        ubuntu:
+          arm64v8: grafana-9.4.17
+          amd64: grafana-9.4.17
+        windows:
+          amd64: node-v18.20.0
+        macos:
+          amd64: http-proxy-0.7.0
+          arm64v8: http-proxy-0.7.0
 
 - case: 'Upgrade: Non vulnerable to non vulnerable'
   id: upgrade_package_nonvulnerable_to_nonvulnerable
   description: |
     Upgrade of a non vulnerable package to non vulnerable
-      macos:
-        Used Package: http-proxy 0.7.2 - npm Format
-        "CVE": [],
-      windows:
-        Used Package: Node 18.20.2 - .msi Format
-        "CVE": [],
-      ubuntu:
-        Used Packages: Grafana 9.5.13 - .deb Format
-        CVE: [],
-      centos:
-        Used Packages: Grafana 9.5.13 - .rpm Format
-        CVE: [],
-  preconditions:
-    tasks:
-      - operation: install_package
-        target: agent
-        check:
-          alerts: true
-          state_index: true
-        package:
-          windows:
-            amd64: node-v18.20.0
-          macos:
-            amd64: http-proxy-0.7.0
-            arm64v8: http-proxy-0.7.0
```

Comment:
Why have the preconditions been deleted?

Reply:
The previous test case `upgrade_package_remove_vulnerability`, is installing the preconditions packages:

 ```
        windows:
          amd64: node-v18.20.0
        macos:
          amd64: http-proxy-0.7.0
          arm64v8: http-proxy-0.7.0
```